src/wfs/src/test/java/org/geoserver/wfs/TransactionTest.java

```diff
@@ -438,4 +439,94 @@ public void testUpdateWithDifferentPrefix() throws Exception {
               .getFirstChild().getNodeValue());
    }
 
+    @Test
+    public void testInsert11Features() throws Exception {
+        String features = "<wfs:Transaction service=\"WFS\" version=\"1.0.0\" "
+                + "xmlns:cgf=\"http://www.opengis.net/cite/geometry\" "
+                + "xmlns:ogc=\"http://www.opengis.net/ogc\" "
+                + "xmlns:wfs=\"http://www.opengis.net/wfs\" "
+                + "xmlns:gml=\"http://www.opengis.net/gml\"> "
+                + "<wfs:Insert > " + "<cgf:Lines>" + "<cgf:lineStringProperty>"
+                + "<gml:LineString>" + "<gml:coordinates decimal=\".\" cs=\",\" ts=\" \">"
+                + "494475.71056415,5433016.8189323 494982.70115662,5435041.95096618"
+                + "</gml:coordinates>" + "</gml:LineString>"
+                + "</cgf:lineStringProperty>" + "<cgf:id>t0002</cgf:id>"
+                + "</cgf:Lines>" + "</wfs:Insert>"
+                + "<wfs:Insert > " + "<cgf:Lines>" + "<cgf:lineStringProperty>"
```

Comment:
You are testing a sequence of wfs:Inserts.

You may wish to test a single wfs:Insert with a feature collection of 11 features?